src/Dependencies/Roslyn/project.json

```diff
@@ -1,7 +1,8 @@
 {
   "supports": {},
   "dependencies": {
-    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09"
+    "Microsoft.VisualStudio.LanguageServices": "2.0.0-beta5-60909-09",
+    "System.ValueTuple": "4.0.1-beta-24425-02"
```

Comment:
@333fred You may want to chat with @agocke. He mentioned the RPS question for adding ValueTuple to VS/Roslyn. I think he's looking into it.